apps/emqx_bridge_kinesis/src/emqx_bridge_kinesis.erl

```diff
@@ -134,12 +167,38 @@ fields("get_producer") ->
 fields("post_producer") ->
     [type_field_producer(), name_field() | fields("config_producer")];
 fields("put_producer") ->
-    fields("config_producer").
+    fields("config_producer");
+fields("config_connector") ->
+    emqx_connector_schema:common_fields() ++
+        connector_config_fields() ++
+        emqx_connector_schema:resource_opts_ref(?MODULE, connector_resource_opts);
+fields(connector_resource_opts) ->
+    emqx_connector_schema:resource_opts_fields();
+fields("put_bridge_v2") ->
+    fields(kinesis_action);
+fields("get_bridge_v2") ->
+    fields(kinesis_action);
+fields("post_bridge_v2") ->
+    fields("post", kinesis, kinesis_action).
```

Comment:
```suggestion
fields(Field) when
    Field == "get_source";
    Field == "post_source";
    Field == "put_source"
->
    emqx_bridge_v2_schema:api_fields(Field, kinesis, fields(kinesis_action)).
```
?
(also, `kinesis` above could be an `?ACTION_TYPE` definition to make it more clear)